### PR TITLE
Update game over message

### PR DESCRIPTION
**Pull request by justinwbaldwin**

### CODE REVIEW

**Review by justinwbaldwin — COMMENTED** [review]
The message displayed after the game ends now includes the instructions to refresh & restart gameplay, and some emojis.
Nice work.

**Review by justinwbaldwin — COMMENTED** [review]
Unclear about how to add myself as a reviewer, as when I tried to add my name, it wouldn't pop up. Instead, I added myself as the assigned person and left my remark as a comment after viewing the files changed. Below is a screenshot of where I was trying to add myself as a reviewer:
image

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (justinwbaldwin) [inline index.html]:
```suggestion
<h3>Are you an artist or a content-creator?<h/3>
```

**Comment by timholy** [comment]
I see a line, "justinbaldwin self-assigned this 2 days ago". LGTM, but let me know if I'm not understanding properly.

**Pull request merged**